### DISCUSSION TITLE
Welcome to tracks Discussions!

### DISCUSSION THREAD

**Discussion by ZeiP** [TracksApp/tracks#2798] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peter2cfu** [comment]
Using tracks version 2.61 on turnkey Linux 17.1 Debian dist, and hit two issues.

1. unable to get tracks to send emails
2. self signup page, seems to be not working, e.g. user cannot self sing up, and as email is not working i don't get the notifications.

I have looked though the very scant documentation on both github and turnkey and can see northing that points me in the right direction, i assume it just becouse im not a key word master or somthing, i'm sure these are common question's.

Passible for any advise on this.

**Comment by SeanKenny64** [comment]
I'm so glad your doing this! Tracks kept me focussed for 10 years between 2005-2015. I'm a bit unfocussed at the moment, I would try your trial but I'm paid up on todoist til next july though
